### PR TITLE
first minor version - when its ready

### PR DESCRIPTION
**Pull request by womullan**

### CODE REVIEW

File: arc.tex [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+
+\begin{enumerate}
+\item Threats and Security infrastructure
+\item Disaster recovery
+\end{enumerate}
+
+\subsection{Prompt \gls{Enclave}}
+
+The Prompt enclave receives images from the Observatory facilities in Chile via a Long Haul Network connection.
```
Comment (frossie) [inline arc.tex]: Should we explicitly refer to Chile as "the summit enclave"?

File: arc.tex [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+\emph{Streak} alerts are transmitted to an \gls{Alert} Vetting System (AVS) located at a Trusted Broker Facility at  Lawrence Livermore National Lab.
+\emph{Non-streak} alerts and \emph{streak} alerts approved by the \gls{AVS} are to be published to the world at large within 60 seconds of the original raw image being taken.
+Normal science images are made available to data rights holders in the \gls{DAC} after an 80 hour embargo period.
+Delayed science images, as identified by the \gls{AVS}, and any unapproved \emph{streak} measurements taken from them are made available after a 10 day embargo period. Commissioning images are made available to data rights holders after a 30 day embargo period.
+
+All Prompt data products are checked for quality by automated systems but also by human operators from the Rubin Observatory staff, who have access to all images and data products in order to perform this function.
```
Comment (frossie) [inline arc.tex]: This could create the impression that humans look at everything. Maybe insert "can be spot-checked"?

File: arc.tex [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+\subsubsection{ Threats and Security infrastructure}
+The obvious threat surfaces here are :
+\begin{enumerate}
+\item Transmission of Data from Chile. IPSec built into the routers will be used on the \gls{LHN}. \citeds{DMTN-108} discusses threats in this realm a little more.
+\item Transmission to \gls{LLNL}.  This will be over internet using TLS.
```
Comment (frossie) [inline arc.tex]: Is this going over some kind of DOE network like ESnet? Cause IPSecing the images to SLAC and then doing all the work to extract the information a bad guy would be most interested in and then using TLS to transmit it to Livermore implies to me that TLS is adequate for image transfer?

File: arc.tex [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+\item Staff access for \gls{QA}. All the usual user threats such as phishing apply - these users are however governed bu SLAC security policies. \action{RD}{March 2022}{ Need ref to SLAC security polices for users .. i.e. FACTS and all that .. }
```
Comment (frossie) [inline arc.tex]: SLAC security policy presumably does't cover summit and base staff. If we are explicitly dealing with post-summit activities maybe make that explicitly clear in the document title.

File: arc.tex [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+\item QA tools. The web accessible QA tools should have a threat analysis although they will be behind SLAC \gls{VPN} and 2FA.
```
Comment (frossie) [inline arc.tex]: Seems a bit passive. A threat analysis would be conducted by who? Also I think QA tools is covered by staff access really, as they are all internal and the people are using the tools.

File: arc.tex [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+All embargoed data is also stored on a secure server in Chile hence it can be retransmitted as needed.
+In the case of a total wipe out of the \gls{OGA} systems use of Chef, docker etc allow redeployment rapidly.
+See also the \gls{SLAC} Rubin disaster recovery plan. \action{RD}{March 22}{Need a \gls{SLAC} disaster recover plan}.
+
+
+\subsection{ Offline Production  \gls{Enclave}}
+Each year (or more frequently), the Offline Production enclave takes the raw images accumulated to date in the Archive and reprocesses them to generate highly accurate, consistent images and measurement catalogs, known as a Data Release. These data products are stored in the Archive and made available to data rights holders in the DAC after they have been checked by automated systems and after Rubin Observatory staff has vetted, characterized, and documented them. Offline Production is split between the USDF and the \gls{FrDF} and UKDF. Each Data Facility performs part of the computations and exchanges its results with the others, so all have a complete set of data products at release time.
+
+
+\subsubsection{ Threats and Security infrastructure}
```
Comment (frossie) [inline arc.tex]:
```suggestion
\subsubsection{ Threats and Security infrastructure}

Offline production data is no longer embargoed ergo not considered under threat. 
```

File: arc.tex [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+\subsubsection{Disaster recovery}
+
+
+\subsection{ \gls{Archive}  Enclave}
+The raw images, data products, and other records of the survey such as commands, events, and telemetry from Observatory systems are all stored in the \gls{Archive}. As the permanent scientific record of the survey, no more than 1% of the raw images or telemetry may be lost or corrupted.
```
Comment (frossie) [inline arc.tex]:
```suggestion
The raw images, data products, and other records of the survey such as commands, events, and telemetry from Observatory systems are all stored in the \gls{Archive}. As the permanent scientific record of the survey, no more than 1\% of the raw images or telemetry may be lost or corrupted according to Rubin requirements. 
```

File: arc.tex [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+
+To help ensure this, the French Data Facility maintains a disaster recovery copy of all raw images and data products. Additional copies of some raw images and data products will be stored in Observatory systems in Chile.
+
+\subsubsection{ Threats and Security infrastructure}
```
Comment (frossie) [inline arc.tex]:
```suggestion
\subsubsection{ Threats and Security infrastructure}

Archived data is no longer embargoed ergo not considered under threat. 
```

File: arc.tex [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+Data rights holders will use the services and systems in this enclave to work with the survey data products.
+It is therefore a general-purpose scientific computing facility. Generally users will interact with the Rubin Science Platform (\gls{RSP}), which is composed of a web-based Portal Aspect providing a guided user interface for accessing and analyzing the data, a Notebook Aspect providing an interactive, flexible, programming-oriented interface, and an API Aspect providing an programmable access service.
+Users of the DAC may connect from anywhere in the world over the Internet; all such users will be authenticated before accessing any \gls{RSP} service.
+The \gls{RSP} is hosted on a cloud service, currently  Google Cloud Platform.
+
+The DAC retrieves the released data products from the \gls{Archive} Enclave via protocols and services authenticated at a service account level only. While end-user identities may be included for audit and accounting purposes, fundamentally the DAC exists to provide access to all \gls{Archive} contents.
+
+\subsubsection{ Threats and Security infrastructure}
+The \gls{RSP} is a major attack surface for the DAC.
```
Comment (frossie) [inline arc.tex]:
```suggestion
The \gls{RSP} is an attractive generic target due to its computing resources but has no confidential data.
```

File: arc.tex [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+Hosting it on a cloud provider reduces risk considerably.
```
Comment (frossie) [inline arc.tex]:
```suggestion
Hosting it on a cloud provider reduces risk considerably for the Archive enclave, and also leverages the security products and services made available by the hosting provider. 
```

File: arc.tex [hunk 11]
```diff
@@ -0,0 +1,110 @@
...
+\citeds{SQR-041} provides a risk assessment for the \gls{RSP}.
+\citeds{DMTN-193} provides a more in depth web risk analysis.
+
+Backend archive services could provide another attack surface.
```
Comment (frossie) [inline arc.tex]: Why is this in the DAC/RSP section?

File: body.tex [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+After an agreed embargo period all data will be transmitted to the \gls{IN2P3} in Lyon, France, and at least some data is transmitted to the Royal Observatory Edinburgh.
+Processing occurs in all three locations using a coordinate release of the Rubin \gls{Science Pipelines}.
+
+Access to data will be provided to authorized via a US \gls{DAC}
+A data access center similar to the \gls{US} \gls{DAC} will also be provided in Chile.
+
+For data taken initially we have been requested to comply with \citeds{NIST.SP.800-171} the compliance matrix is provided in \appref{ap:compliance}
+
+
+This document does not do a detailed threat analysis though this should be and are done for individual systems and \gls{Enclave}s. \cite{TCISSG} is good guide for software threats.
```
Comment (frossie) [inline body.tex:32]: 1. Not sure what that TCISSG citation is
2. I think this risks saying that the plan is to have a plan. Maybe we can define a rubrik for each enclave to describe its arrangements.

**Review by frossie — COMMENTED** [review]
Did not review the compliance document in detail; as long as it's about the embargoed enclaves and not eg. the RSP in general, should be okay.

File: arc.tex (hunk 2 above)
Comment (ktlim) [reply arc.tex]: Sure.  Most likely they'll monitor summary statistics and spot-check random samples.  The main point here was that staff need to have access to anything at any time, not pre-selected samples or something like that.

File: arc.tex (hunk 3 above)
Comment (ktlim) [reply arc.tex]: It would be ESnet, but beyond that it could be that a physical network on US soil is harder to infiltrate.

File: arc.tex (hunk 1 above)
Comment (ktlim) [reply arc.tex]: Data will also flow (physically, if not logically) through equipment at the Base.

File: arc.tex (hunk 9 above)
Comment (ktlim) [reply arc.tex]: User-generated data is still sensitive.

File: arc.tex (hunk 4 above)
Comment (womullan) [reply arc.tex]: not sure i understand but there are new sections now for summit and control etc ..

File: arc.tex (hunk 5 above)
Comment (womullan) [reply arc.tex]: ok epxlicit and noted its staff

File: arc.tex (hunk 6 above)
Comment (womullan) [reply arc.tex]: good yes added

File: arc.tex (hunk 8 above)
Comment (womullan) [reply arc.tex]: yes

File: arc.tex (hunk 9 above)
Comment (womullan) [reply arc.tex]: an attractive generic target due to its computing resources.
There is some user genereated data which is mildly sensitive.

File: arc.tex (hunk 11 above)
Comment (womullan) [reply arc.tex]: back end services are part of the DAC right ?

File: body.tex (hunk 12 above)
Comment (womullan) [reply body.tex:32]: ok we can discuss - its pretty endless though

File: arc.tex (hunk 3 above)
Comment (womullan) [reply arc.tex]: We did try to have TLS also from Chile but OGAs said not

**Pull request merged**